src/components/Community/AddPost/PostMatchReview.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+interface MatchReviewProps {
+  onSetReview: (review: string) => void;
+}
+
+const PostMatchReview: React.FC<MatchReviewProps> = ({ onSetReview }) => {
```

Comment:
컴포넌트 타입은 FC를 사용하지 않고 이렇게 잡아주시면 좋을 것 같아요!
```suggestion
const PostMatchReview = ({ onSetReview }: MatchReviewProps) => {
```